playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -18,6 +18,15 @@
 ## BEGIN RACCOONGANG ##
 EDXAPP_S3_USE_SIGV4: false
 EDXAPP_S3_HOST: null
+
+EDXAPP_THIRD_PARTY_AUTH_BACKENDS:
+ - social_core.backends.google.GoogleOAuth2
+ - social_core.backends.linkedin.LinkedinOAuth2
+ - social_core.backends.facebook.FacebookOAuth2
+ - social_core.backends.azuread.AzureADOAuth2
+ - social_core.backends.vk.VKOAuth2
+ - third_party_auth.saml.SAMLAuthBackend
+ - third_party_auth.lti.LTIAuthBackend
```

Comment:
Why do we need this?

https://github.com/edx/edx-platform/blob/dcb77ff7aeedab26c69b91173dfc8122dd2e63da/lms/envs/aws.py#L682-L690